sections/production/installpackageswithnpmci.md

```diff
@@ -0,0 +1,30 @@
+# Install packages with npm ci in production
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+You locked your dependencies following [**Lock dependencies**](/sections/production/lockdependencies.md) but you now need to make sure those exact package versions are used in production.
+
+Using `npm ci` to install packages will do exactly that and more.
+* It will only install packages in the `dependencies` section of your `package.json`. `devDependencies` are ignored
+* It will fail if your `package.json` and your `package-lock.json` do not match (they should) or if you don't have a lock file
+* If a `node_modules` folder is present it will automatically remove it before installing
+* It is faster! Nearly twice as fast according to the release blog post (see link below)
+
+As per the first point, it's good to note that `devDependencies` have nothing to do on a production server (hence the name).
+It also reduces the size of the bundle of code you are shipping (for example inside a docker image).
+
```

Comment:
Makes it even more tangible, show an example of when this can happen:
1. Someone changes manually package.json, not through a cli command but rather direct package.json editing. Then a gap between package.json & .lock is presented
2. An attacker changes only the lock to refer to a malicious package/version (becauase no one review these huge files), if that malicious dep won't exist in package.json the command will fail (I think)

Reply:
I checked, the second point doesn't hold (and yet I hoped it would 😞 ) You have to check that by manually looking through the dependency list.
I also found https://github.com/lirantal/lockfile-lint but it seems that if the malicious package is hosted on npm.js you're done.